### PR TITLE
Fix failed hash verification #795

### PR DESCRIPTION
**Pull request by wilsonvargas**
fix issue #795 

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/796)

### CODE REVIEW

**Comment by chausner** [comment]
This does not fix the underlying issue. The real problem is that the download link, even though it contains a version number, actually points to the latest version, and there has been an update published during the last couple of days.

This should be the correct download link for 7.24.0: https://dl.pstmn.io/download/version/7.24.0/win64. And since 7.25.0 is out now, this is the latest one: https://dl.pstmn.io/download/version/7.25.0/win64

File: manifests/Postman/Postman/7.24.0.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ Homepage: https://www.postman.com/
 Installers:
   - Arch: x64
     Url: https://dl.pstmn.io/download/latest/win64?filename=Postman-win64-7.24.0-Setup.exe
```
Comment (jamierocks) [inline manifests/Postman/Postman/7.24.0.yaml:12]: Should be `https://dl.pstmn.io/download/version/7.24.0/win64` as @chausner noted.

**Review by jamierocks — CHANGES_REQUESTED** [review]

**Comment by Bodup** [comment]
I've made a [pull request](https://github.com/microsoft/winget-pkgs/pull/769) that should prevent these sort of issues in the future.

**Comment by jamierocks** [comment]
As this PR was closed, I've opened https://github.com/microsoft/winget-pkgs/pull/819 to fix the download for 7.24.0 - in addition to my PR for 7.25.0 :)

**Pull request closed without merging**